docker-integrations/tendermint/README.md

```diff
@@ -0,0 +1,227 @@
+# @kyvejs/tendermint
+
+## Content
+
+- [Introduction](#introduction)
+- [Use cases](#use-cases)
+- [Required Setup](#required-setup)
+- [Docker Setup](#docker-setup)
+  - [Docker assumptions](#docker-assumptions)
+- [Integrations currently live](#integrations-currently-live)
+  - [Mainnet](#mainnet)
+  - [Testnet](#testnet)
+  - [Devnet](#devnet)
+- [Binary Installation](#binary-installation)
+  - [Build from source](#build-from-source)
+  - [Download prebuilt binary](#download-prebuilt-binary)
+- [Run a node](#run-a-node)
+- [Creating a pool with the runtime](#creating-a-pool-with-the-runtime)
+  - [Config](#config)
+  - [Create Pool governance proposal](#create-pool-governance-proposal)
+
+## Introduction
+
+This runtime validates and archives blocks from any tendermint based blockchain application. It only stores blocks from and to a given height and makes
+them available to directly download them from the storage provider, or directly sync them into the blockchain nodes itself, therefore enabling block sync
+via validated KYVE blocks.
+
```

Comment:
@kchrist-rocketfueldev Can you also add a section that briefly describes the architecture of the containerized solution? That should mention the gRPC server and the total encapsulation of a runtime in a single image.

Reply:
Added something simplistic to describe the architecture